src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -13,6 +13,8 @@ internal class LocalizableStrings
 
         public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to add.";
 
+        public const string CmdPackageDescription = "Name of the package to be added.";
```

Comment:
There is not a capability change, only an improvement to cmd line parsing. Let's revert the string changes for 1.0.0, or bring them as a separate bug.